components/Snackbar.vue

```diff
@@ -16,13 +16,26 @@ export default Vue.extend({
     };
   },
   created() {
+    // Empties potential already set buffer
+    this.displaySnackbar(
+      this.$store.state.snackbar.message,
```

Comment:
`computed: mapState<AppState>({
   snackBar: (state) => state.snackBar
})`

Reply:
`this.snackBar.message, this.snackBar.color`

Reply:
`computed: mapState<AppState>({
    snackBar: (state: AppState) => state.snackBar
  })` doesn't work, the `snackBar` computed prop in the component is undefined

Reply:
Do we actually need that exactly in the `created` lifecycle hook?

Reply:
Maybe in `mounted` we could do it, I should try.

Reply:
Done, it works in mounted

Reply:
You can use ‘mapState’ I guess now?